x-pack/plugins/maps/common/constants.ts

```diff
@@ -30,6 +30,7 @@ export const CHECK_IS_DRAWING_INDEX = `/${GIS_API_PATH}/checkIsDrawingIndex`;
 
 export const MVT_GETTILE_API_PATH = 'mvt/getTile';
 export const MVT_GETGRIDTILE_API_PATH = 'mvt/getGridTile';
+export const OPEN_WIZARD_URL = 'openLayerWizard';
```

Comment:
rename `OPEN_WIZARD_URL` to `OPEN_LAYER_WIZARD`. Its not a URL so it should not have URL in the name